recipes/uwsgi/bld.bat

```diff
@@ -0,0 +1,6 @@
+set UWSGI_PROFILE=%RECIPE_DIR%/uwsgi_config.ini
+set UWSGI_INCLUDES=%PREFIX%/include,%PREFIX%/include/openssl
+set LDFLAGS=-L%PREFIX%/lib
+
+python setup.py install

```

Comment:
We also need to add these arguments (`--single-version-externally-managed --record record.txt`) here too.